### PR TITLE
Add basic components

### PR DESCRIPTION
**Pull request by KaushikShivam**
- Add a component directory. It has 
   - App.js (Renders Display and ButtonPanel)
   - Button.js (accepts a name prop which is a string)
   - ButtonPanel.js (renders all the Button components)
   - Display.js (Displays the calculation string as a prop, default is 0)

### CODE REVIEW

File: src/components/Button.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+import './Button.css';
+
+const Button = ({ name }) => {
+  return <button className='Button'>{name}</button>;
+};
+
```
Comment (SunnyAsar) [inline src/components/Button.js]: Hey, please use propTypes to validate what is coming here for name is a string

File: src/components/Display.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  render() {
+    const { result } = this.props;
+    return (
+      <div className='Display'>
+        <h2>Result: {result}</h2>
+      </div>
+    );
+  }
+}
+
```
Comment (SunnyAsar) [inline src/components/Display.js]: let's use propTypes here as well.

File: src/components/Button.js (hunk 1 above)
Comment (KaushikShivam) [reply src/components/Button.js]: Hello. I've done the needful.

File: src/components/Display.js (hunk 2 above)
Comment (KaushikShivam) [reply src/components/Display.js]: I've done the needful.

**Comment by KaushikShivam** [comment]
I'll hold off taking style files to another directory as I'm thinking of implementing the directory structure like this. It's from a course I'm following which suggests that this is the industry standard.
Screen Shot 2019-11-06 at 6 24 35 PM

**Comment by SunnyAsar** [comment]
well @KaushikShivam  i don't know about that style being a standard but i think it can make your structure huge fast as each component now needs a folder. take a look at styled components, you may like it, it allow you to write your styles for a component inside the component using regular css or sass.

**Comment by KaushikShivam** [comment]
> well @KaushikShivam i don't know about that style being a standard but i think it can make your structure huge fast as each component now needs a folder. take a look at styled components, you may like it, it allow you to write your styles for a component inside the component using regular css or sass.

@SunnyAsar - Yes Yes. I know. I wasn't talking about the styles. I was talking about the directory structure being the standard. I've already worked on a project using JSS which is picking up quite fast and shares the same idealogy as Styled components. Will start learning styled components pretty soon as well. Thanks :)

**Pull request merged**